### ISSUE TITLE
"optional" property of ServerOptions is not optional

### ISSUE THREAD

**Issue by hal-mat** [HC200ok/vue3-easy-data-table#59]
## TL;DR
In this page, every ServerOptions properties are optional. But "page" and "rowsPerPage" are required. 
https://hc200ok.github.io/vue3-easy-data-table-doc/props/server-side-mode-only-props.html

## how to cause
`
const serverOptions = ref<ServerOptions>({
});
`

## expected
Call api with no parameter.

## error
`TS2769: No overload matches this call.`
`Overload 1 of 3, '(value: ServerOptions): Ref<{ page: number; rowsPerPage: number; sortBy?: string | undefined; sortType?: SortType | undefined; }>', gave the following error.  `
`Argument of type '{}' is not assignable to parameter of type 'ServerOptions'.`
`Type '{}' is missing the following properties from type 'ServerOptions': page, rowsPerPage`
`Overload 2 of 3, '(value: ServerOptions): Ref<{ page: number; rowsPerPage: number; sortBy?: string | undefined; sortType?: SortType | undefined; }>', gave the following error.`
`Argument of type '{}' is not assignable to parameter of type 'ServerOptions'.`
`Type '{}' is missing the following properties from type 'ServerOptions': page, rowsPerPage`

3rd error is unrelated.

## version
vue@3.2.37
vue3-easy-data-table@1.2.28
typescript@4.7.4

**Comment by HC200ok** [comment]
@hal-mat Sorry for the wrong description in the document, I have updated this page: https://hc200ok.github.io/vue3-easy-data-table-doc/props/server-side-mode-only-props.html 
Thank you for your report!
